common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -335,6 +323,118 @@
                 });
 
             });
+    </script>
+}
+
+@bootSystemJs() = {
+
+    <script>
+        @Html(Static.js.es6ModuleLoader)
+        @Html(Static.js.systemJs)
+        @Html(Static.js.systemJsAppConfig)
+
+        @if(!play.Play.isDev()) {
+            @Html(Static.js.systemJsBundleConfig)
+        }
+
+        System.paths['ophan/ng']  = '@{Configuration.javascript.config("ophanJsUrl")}.js';
+        System.paths['googletag'] = '@{Configuration.javascript.config("googletagJsUrl")}';
```

Comment:
just because the configs are just string properties on file, and one has .js, the other doesn't